### ISSUE TITLE
Cannot build storybook

### ISSUE THREAD

**Issue by stevepiercy** [plone/volto#5479]
```
$ make storybook-build

> @plone/volto@18.0.0-alpha.3 build-storybook /Users/stevepiercy/projects/Plone/documentation/submodules/volto/packages/volto
> build-storybook "-o" "docs/_build/storybook"

sh: build-storybook: command not found
 ELIFECYCLE  Command failed.
 WARN   Local package.json exists, but node_modules missing, did you mean to install?
make: *** [storybook-build] Error 1
```

A similar error occurs in documentation.

```
$ make storybook
cd submodules/volto/packages/volto && pnpm build-storybook -o ../../../../_build/html/storybook

> @plone/volto@18.0.0-alpha.3 build-storybook /Users/stevepiercy/projects/Plone/documentation/submodules/volto/packages/volto
> build-storybook "-o" "../../../../_build/html/storybook"

sh: build-storybook: command not found
 ELIFECYCLE  Command failed.
 WARN   Local package.json exists, but node_modules missing, did you mean to install?
make: *** [storybook] Error 1
```

**Comment by stevepiercy** [comment]
This issue also requires that the documentation build of the storybook be re-enabled and Makefile updated with new commands and paths. I think that this would be correct for documentation:

```makefile
.PHONY: storybook
storybook:
	cd submodules/volto/packages/volto && pnpm i && pnpm build-storybook -o ../../../../_build/html/storybook
```

**Comment by sneridagh** [comment]
@stevepiercy no, it would not be correct, because that makefile has to run locally too.

**Comment by sneridagh** [comment]
The PR should have worked, probably the tip needed to be updated.

**Comment by stevepiercy** [comment]
Closed by #5482. See also https://github.com/plone/documentation/pull/1588 and subsequent commits. https://6.docs.plone.org/storybook/
